docs/src/man/split_apply_combine.md

```diff
@@ -63,11 +67,31 @@ julia> by(iris, :Species, df -> DataFrame(N = size(df, 1)))
 │ 3   │ virginica     │ 50    │
 ```
 
-The `by` function also support the `do` block form:
+julia> by(iris, :Species, N = :Species => length, mean = :PetalLength => mean) # Column for length is arbitrary
```

Comment:
same here - actually you can pass more than one keyword argument - this is not mentioned (and what is allowed to be passed multiple times). And I think it should be clarified if mixing styles is allowed.